spec/models/comment_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  it 'is valid with valid attributes' do
+    user = User.create(name: 'John Doe')
+    post = Post.create(author: user, title: 'My Post', text: 'This is my post content.')
+    comment = Comment.new(user:, post:, text: 'Great post!')
+    expect(comment).to be_valid
+  end
+
+  it 'is not valid without text' do
+    comment = Comment.new(text: nil)
+    expect(comment).not_to be_valid
```

Comment:
- [ OPTIONAL ] Great job for adding the tests for the post validation but I think it would be great if you add tests for the methods here as mentioned in the requirements to test all the methods so please test increment_post_comments_counter that in your comment model 🔥⭐♻
 
 
![image](https://github.com/ReemMohamedAbdelfatah/rails-blog-app/assets/58553711/84b9a87b-5798-4819-8960-e3285d69283d)